marianas_process_loop_umi_1.8.0/marianas_process_loop_umi.cwl

```diff
@@ -0,0 +1,95 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  sbg: 'https://www.sevenbridges.com/'
+id: marianas_process_loop_umi_cwl
+baseCommand:
+  - java8
+inputs:
+  - id: fastq1
+    type: File?
+    inputBinding:
+      position: 1
+  - id: fastq2
+    type: File?
+  - id: sample_sheet
+    type: File?
+  - id: umi_length
+    type: int?
+    inputBinding:
+      position: 2
+  - id: input
+    type: File?
+  - id: output_project_folder
```

Comment:
good point, the tool has this option in order to specify where to write results, but for CWL purposes we only specify it as `.`

https://github.com/mskcc/Marianas/blob/master/src/org/mskcc/marianas/umi/duplex/DuplexUMIBamToCollapsedFastqFirstPass.java#L56

Reply:
@sivkovic can you please review this and merge it

Reply:
Can we just remove `output_project_folder` as an input, and move it to the arguments with value `.`, since if we keep it as a input, someone can change the value, and it can cause the tool to fail.

Reply:
we removed this arg from the tools so we should be all good here now